include/onnxruntime/core/optimizer/graph_transformer_level.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+
+namespace onnxruntime {
+
+enum class TransformerLevel : uint32_t {
+  Default = 0,
```

Comment:
I always wonder why people put = 0 in these since the C++ standard requires all enums start at zero if you don't override it :) (C++ standard section 7.2 - Enumeration declarations [dcl.enum])